src/server.js

```diff
@@ -39,7 +39,12 @@ class Server extends EventEmitter {
 	}
 
 	parse(message, rinfo) {
-		const packet = NTPPacket.parse(message);
+		let packet;
+		try {
+			packet = NTPPacket.parse(message);
+		} catch (e) {
+			return
+		}
```

Comment:
I can't approve this since the invalid package should be a break point for the library. If you don't want it to break on an invalid package error, you should treat the exception in your application code.

```suggestion
		let packet;
		try {
			packet = NTPPacket.parse(message);
		} catch (e) {
			throw new Error(e)
		}
```

Reply:
If we throw exception in that method the app crashed on invalid packet.
How can we catch this exception in app?

Reply:
Using a try/catch block aroung the call of the syncTime method. The package itself cannot ignore an invalid packet error, it should by pattern definition throw when an invalid NTP packet comes to it.

Reply:
Server does not have syncTime method.
Did you try to send invalid packet? App will crash.
Mb we need to add `invalid_packet` event instead of just return void?

Reply:
Actually what it does need is a way of clearly seeing the exception happening but without breaking the whole server. An event is maybe not a clear way of handling this.

I think that if we added a error handling callback at the instatiation of the class and make it's default value be a function that just logs the incoming error would be best. Treating it like this should make the debug process easier and shouldn't make the library denser on documentation and harder on understanding.